imod/mf6/wel.py

```diff
@@ -248,7 +320,7 @@ def _create_cellid(
         df_for_cellid = assigned_wells.groupby("index").first()
         d_for_cellid = df_for_cellid[["x", "y", "layer"]].to_dict("list")
```

Comment:
I find the name cellid confusing. In MODFLOW terms this is the row, column, layer or the 2dcellid, layer.
In this case it would call it something like `id`, `identifier`, `name` or `well_id`

Reply:
But row, column, layer is what I refer to with the ``cellid``. At some point in the future we might also have to support 2dcellid, but that will probably not happen soon.